app/models/backend_api_config.rb

```diff
@@ -11,6 +11,10 @@ class BackendApiConfig < ApplicationRecord
   has_one :proxy, through: :service
   has_many :proxy_rules, through: :proxy
 
+  after_create do
```

Comment:
Is this part of this PR?

Reply:
This is because you can choose an existing backend API instead of creating a new one.